torch/_subclasses/fake_tensor.py

```diff
@@ -656,25 +695,31 @@ def __new__(cls, fake_mode, elem, device, constant=None, real_tensor=None):
     # In particular, the __init__ call will look funny in the following case:
     #
     #   with FakeTensorMode():
-    #       x = torch.Tensor([1, 2, 3])
+    #       x = Tensor([1, 2, 3])
     #
     # this desugars into:
     #
     #   with FakeTensorMode():
-    #       x = torch.Tensor.__new__([1, 2, 3])
+    #       x = Tensor.__new__([1, 2, 3])
     #       # NB: x is a fake tensor, because of the mode!
     #       x.__init__([1, 2, 3])  # not the normal fake tensor args!
     #
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args: object, **kwargs: object) -> None:
```

Comment:
nit: I think the convention is typically `Any` over `object`, also applies to other : object annotations

Reply:
`Any` is a poisoning type - meaning that you lose all type checking on anything that touches it.  For example:

```
def my_str_fn(x: str) -> int: ...
```
If you have `*args:Any` then doing:
```
my_str_fn(args[0])
```
gets no complaint, whereas if you have `*args:object` then it will complain that `args[0]` isn't a `str`.